CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- Case-sensitive treatment of file names in coverage reports on Windows.
```

Comment:
This isn't accurate anymore - maybe something like "Incorrect case sensitive treatment of unit names in coverage reports"? Or could be rephrased to emphasise the problem rather than the root cause, e.g. "Units being mapped case-sensitively to case-insensitive coverage data." Up to you.